app/src/androidTest/java/com/github/se/cyrcle/model/user/UserTestInstances.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.se.cyrcle.model.user
+
+/*
+// Mock Locations
+val mockLocation1 =
+    Location(
+        topLeft = Point.fromLngLat(-73.935242, 40.730610), // Top-left corner (NYC)
+        topRight = Point.fromLngLat(-73.925242, 40.730610), // Top-right corner (NYC)
+        bottomLeft = Point.fromLngLat(-73.935242, 40.720610), // Bottom-left corner (NYC)
+        bottomRight = Point.fromLngLat(-73.925242, 40.720610) // Bottom-right corner (NYC)
+    )
+
+val mockLocation2 =
+    Location(
+        topLeft = Point.fromLngLat(-0.1276, 51.5074), // Top-left corner (London)
+        topRight = Point.fromLngLat(-0.1176, 51.5074), // Top-right corner (London)
+        bottomLeft = Point.fromLngLat(-0.1276, 51.4974), // Bottom-left corner (London)
+        bottomRight = Point.fromLngLat(-0.1176, 51.4974) // Bottom-right corner (London)
+    )
+*/
+
+// Mock Users
+val user1 =
```

Comment:
You could place the `val`s in a `TestInstancesUser` `object`

Reply:
Agreed